llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -984,7 +984,8 @@ def __init__(
                 from flash_attn.losses.cross_entropy import \
                     CrossEntropyLoss as FusedCrossEntropyLoss
 
-                self.loss_fn = FusedCrossEntropyLoss(ignore_index=-100)
+                self.loss_fn = FusedCrossEntropyLoss(ignore_index=-100,
```

Comment:
Outside the scope of this pr but should we make expose `ignore_index` in the config (default `-100`)?
cc @dakinggg

Reply:
I don't think so. Everyone uses -100, not sure why anyone would want to change this

Reply:
resolving cuz outside the scope of this PR